services/libreTranslate.js

```diff
@@ -0,0 +1,35 @@
+const fetch = require("node-fetch");
+
+const mirrors = [
+    "https://translate.terraprint.co/translate",
+    "https://trans.zillyhuhn.com/translate",
+]
+
+const libreTranslate = async (text, {from, to}) => {
+    for await (url of mirrors) {
+        try {
+            const res = await fetch(url, {
+                method: "POST",
+                body: JSON.stringify({
+                    q: text,
+                    source: from,
+                    target: to,
+                    format: "text",
+                }),
+                headers: { "Content-Type": "application/json" }
+            }).then((res) => res.json());
+            
+            return { text: res.translatedText }
+        } catch (err) {
```

Comment:
In these cases, why not log AND throw the error? If not, the use just will wonder what happened

Reply:
Thanks. Added the throw error and error.message to the console.log!. Will improve error handling after adding some color or emojis on console so will look cleaner. But will threat it on another PR.